### PR TITLE
Updated ui of offer successfully created.

### PR DESCRIPTION
**Pull request by iPR0D1G1**
Fixed: #6488

### CODE REVIEW

File: apps/catalog/catalog/src/app/marketplace/title/selection/congratulations/congratulations.component.scss [hunk 1]
```diff
@@ -4,12 +4,17 @@ section {
...
     width: 300px;
   }
-}
\ No newline at end of file
+}
+
+.offer_and_deals{
+  padding: unset;
+  text-decoration: underline;
+}
```
Comment (pldespaigne) [inline apps/catalog/catalog/src/app/marketplace/title/selection/congratulations/congratulations.component.scss:20]: Please ask confirmation to @GrandSchtroumpf but we should not try to style link
It should be done at theme level

**Review by pldespaigne — CHANGES_REQUESTED** [review]

File: apps/catalog/catalog/src/app/marketplace/title/selection/congratulations/congratulations.component.html [hunk 2]
```diff
@@ -1,8 +1,11 @@
 <section fxLayout="column" fxLayoutAlign="center center" fxLayoutGap="16px">
```
Comment (GrandSchtroumpf) [inline apps/catalog/catalog/src/app/marketplace/title/selection/congratulations/congratulations.component.html:1]: remove this section, and apply the style directly on the `:host`

File: apps/catalog/catalog/src/app/marketplace/title/selection/congratulations/congratulations.component.html [hunk 3]
```diff
@@ -1,8 +1,11 @@
...
-  <h1>Your Offer was successfully sent.</h1>
+  <h1>Your Contract Offer <br/> has been successfully sent.</h1>
   <p>
-    Our team will get back to you as soon as possible to coordinate the deal.<br />
-    Feel free to <a [routerLink]="['./']" (click)="openIntercom()">contact us</a> if you have a question.</p>
+    You can always find your contract offer on <a mat-button [routerLink]="['../../offer']" class="offer_and_deals"><strong>Offer & Deals</strong></a> page. <br />
```
Comment (GrandSchtroumpf) [inline apps/catalog/catalog/src/app/marketplace/title/selection/congratulations/congratulations.component.html:4]: Inside a `<p>` we don't apply the `mat-button` on a `a` tag.
Also, if you don't have any variable use the `routerLink` without `[]`.
So it should look like taht
```html
<a routerLink="../../offer">...</a>
```

File: apps/catalog/catalog/src/app/marketplace/title/selection/congratulations/congratulations.component.html [hunk 4]
```diff
@@ -1,8 +1,11 @@
...
+    Feel free to contact Archipel Content Team if you have a question.
+  </p>
+  <button (click)="openIntercom()" mat-button>Contact Archipel Team</button>
   <img asset="contract_offer.svg" alt="Offer has been created successfully">
-  <button mat-flat-button color="accent" [routerLink]="['/c/o/marketplace/title']">Return to Browsing</button>
+  <a mat-flat-button color="primary" [routerLink]="['../../offer']">See Offers & Deals</a>
```
Comment (GrandSchtroumpf) [inline apps/catalog/catalog/src/app/marketplace/title/selection/congratulations/congratulations.component.html:9]: same as above for the `routerLink`

**Review by GrandSchtroumpf — APPROVED** [review]

File: apps/catalog/catalog/src/app/marketplace/title/selection/congratulations/congratulations.component.scss [hunk 5]
```diff
@@ -4,12 +4,16 @@ section {
...
   }
 
   img {
     width: 300px;
   }
-}
\ No newline at end of file
+}
+
+.offer_and_deals{
```
Comment (GrandSchtroumpf) [inline apps/catalog/catalog/src/app/marketplace/title/selection/congratulations/congratulations.component.scss:17]: remove this class

**Review by pldespaigne — APPROVED** [review]

File: apps/catalog/catalog/src/app/marketplace/title/selection/congratulations/congratulations.component.scss [hunk 6]
```diff
@@ -1,15 +1,26 @@
...
 
-  img {
-    width: 300px;
+h1 {
+  text-align: center;
+}
+
+p {
+  text-align: center;
+  a {
```
Comment (GrandSchtroumpf) [inline apps/catalog/catalog/src/app/marketplace/title/selection/congratulations/congratulations.component.scss:19]: you shouldn't have to unset. You should remove it.

**Review by GrandSchtroumpf — APPROVED** [review]

**Pull request merged**